packages/@storefront/flux-capacitor/src/core/actions/payloads.ts

```diff
@@ -2,9 +2,25 @@ import Configuration from '../configuration';
 import Store from '../store';
 
 namespace Payload {
+  export namespace History {
+    export interface State extends ProtoState {
+      method: Function;
+      request?: any;
+    }
+
+    export interface ProtoState {
+      // TODO: add in the future
```

Comment:
Why do we need this if we already have a type definition? https://github.com/groupby/storefront/pull/28/files/d83d76ffaa7c8e38b7acd3d27aa11c0c3ca7dcdc#diff-df780f9863ab20aba10f58f77ab14c83R35 I'm wary about putting external links that could go out of date. The point is that calling the `pushState` method has to match the type definition that we say it matches, it doesn't necessarily have to do what the `window.history.pushState` method does, but it does have to have the right definition.
I'm going to take out the `title` stuff for now since it's not implemented, but yes it would be to call the `pushState` and `replaceState` methods with

Reply:
Noted, withdrawn.